src/apps/frontend/hooks/login-form.hook.ts

```diff
@@ -0,0 +1,50 @@
+import { useFormik } from 'formik';
+import { useState, useCallback } from 'react';
+import * as Yup from 'yup';
+
+import constant from '../constants';
+import { AccessService } from '../services';
+
+const useLoginForm = () => {
+  const [success, setSuccess] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
Recommend creating [this](https://teams.microsoft.com/l/message/19:c388dc298f0f432bb4805cadc4ffe92f@thread.tacv2/1705932812553?tenantId=79836b2a-53cc-4854-81b4-ba2d7c9f2726&groupId=c0e79bd0-36ec-4b10-8be7-55f5b3d2395c&parentMessageId=1705932812553&teamName=Jalan%20Technologies&channelName=Engineering&createdTime=1705932812553) abstraction

Reply:
Will try to come up with it, but for now resuming on Valon's tasks.